domains/transfer/Transfer.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs

```diff
@@ -8,34 +8,31 @@
 using System.Security.Cryptography;
 using System.Text;
 using System.Threading.Tasks;
-using Asp.Versioning.ApiExplorer;
 using API.Transfer.Api.Services;
 using API.Transfer.TransferAgreementProposalCleanup;
+using Asp.Versioning.ApiExplorer;
 using DataContext;
-using DataContext.Models;
 using EnergyOrigin.ActivityLog;
 using EnergyOrigin.ActivityLog.HostedService;
 using EnergyOrigin.TokenValidation.Utilities;
 using EnergyOrigin.TokenValidation.Values;
+using FluentAssertions;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Mvc.Testing;
 using Microsoft.AspNetCore.TestHost;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-using Npgsql;
+using NSubstitute;
 using Testcontainers.PostgreSql;
+using WireMock.Server;
 using Xunit;
 
 namespace API.IntegrationTests.Factories;
 
-public class TransferAgreementsApiWebApplicationFactory : WebApplicationFactory<Program>, IAsyncLifetime
```

Comment:
Why remove the IAsyncLifetime?

Reply:
Because it is controlled by the fixture.